.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
you don't need this file in this project - you just need to have a writer to create it with your script.